src/main/java/uk/gov/hmcts/reform/hmc/model/LinkType.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.hmc.model;
+
+import lombok.Getter;
+
+@Getter
+public enum LinkType {
+
+    IND("Ordered"),
+    ORG("Same Slot");
```

Comment:
Looks like a copy/paste error?
```suggestion
    ORDERED("Ordered"),
    SAME_SLOT("Same Slot");
```

Reply:
:)